qutip/qip/device/modelprocessor.py

```diff
@@ -184,7 +143,7 @@ def run_state(self, rho0=None, analytical=False, qc=None, states=None,
             and then calculate the evolution.
 
         states: :class:`qutip.Qobj`, optional
-         Old API, same as rho0.
+         Old API, same as init_state.
```

Comment:
I understand this will be removed later on.

Reply:
This is an old API. I think I will just remove it as we already broke some default use of the old `Processor` anyway.

Reply:
Deprecation warning exists